src/test/regress/expected/multi_schema_support.out

```diff
@@ -107,3 +108,412 @@ SELECT COUNT(*) FROM "nation._'append";
+           5 | ETHIOPIA                  |           1 | ven packages wake quickly. regu
+           7 | GERMANY                   |           4 | 
+           0 | ALGERIA                   |           1 |  haggle. carefully final deposits detect slyly agai
+           3 | CANADA                    |           2 | eas hang ironic, silent packages. slyly regular packages are furiously over the tithes. fluffily bold
+           4 | EGYPT                     |           5 | y above the carefully unusual theodolites. final dugouts are quickly across the furiously regular d
+           6 | FRANCE                    |           4 | 
+           2 | BRAZIL                    |           2 | y alongside of the pending deposits. carefully special packages are about the ironic forges. slyly special 
+(8 rows)
+
+--test with search_path is set
+SET search_path TO test_schema_support;
+SELECT master_modify_multiple_shards('UPDATE nation_hash SET n_regionkey = n_regionkey + 1');
+ master_modify_multiple_shards 
+-------------------------------
+                             8
+(1 row)
+
+--verify master_modify_multiple_shards
+SELECT * FROM nation_hash;
+ n_nationkey |          n_name           | n_regionkey |                                                  n_comment                                                  
+-------------+---------------------------+-------------+-------------------------------------------------------------------------------------------------------------
+           1 | ARGENTINA                 |           3 | al foxes promise slyly according to the regular accounts. bold requests alon
+           5 | ETHIOPIA                  |           2 | ven packages wake quickly. regu
+           7 | GERMANY                   |           5 | 
+           0 | ALGERIA                   |           2 |  haggle. carefully final deposits detect slyly agai
+           3 | CANADA                    |           3 | eas hang ironic, silent packages. slyly regular packages are furiously over the tithes. fluffily bold
+           4 | EGYPT                     |           6 | y above the carefully unusual theodolites. final dugouts are quickly across the furiously regular d
+           6 | FRANCE                    |           5 | 
+           2 | BRAZIL                    |           3 | y alongside of the pending deposits. carefully special packages are about the ironic forges. slyly special 
+(8 rows)
+
+--test COLLATION with schema
+SET search_path TO public;
+CREATE COLLATION test_schema_support.english FROM "en_US";
+-- create COLLATION in worker node 1 in schema
+\c - - - :worker_1_port
+CREATE COLLATION test_schema_support.english FROM "en_US";
+-- create COLLATION in worker node 2 in schema
+\c - - - :worker_2_port
+CREATE COLLATION test_schema_support.english FROM "en_US";
+\c - - - :master_port
+SELECT n_name = 'Turkey' COLLATE test_schema_support.english FROM test_schema_support.nation_hash;
```

Comment:
This query does not make sense that much. Could you use a typical COLLATE query? Such as using order by
https://www.postgresql.org/docs/9.5/static/collation.html